packages/lesswrong/server/scripts/makeMigrations.ts

```diff
@@ -116,12 +96,6 @@ type Artifact = CollectionNameString | PostgresExtension | PostgresFunction;
  * - `accepted_schema.sql`: This is a SQL view of the schema that has been accepted.
  * - `schema_to_accept.sql`: If the current schema is not accepted, this file will be generated to contain a SQL view of the "unaccepted" schema.
  *   This is useful for comparing against the accepted schema to see what changes need to be made in the migration that is generated. It is automatically deleted when the schema is accepted.
- *
- * @param {boolean} writeSchemaChangelog - If true, update the schema_changelog.json file before checking for changes
- * @param {boolean} writeAcceptedSchema - If true, update the `accepted_schema.sql` and `schema_to_accept.sql`
- * @param {boolean} generateMigrations - If true, generate a template migration file if the schema has changed
- * @param {boolean} rootPath - The root path of the project, this is annoying but required because this script is sometimes run from the server bundle, and sometimes from a test.
- * @param {boolean} forumType - The optional forumType to switch to
```

Comment:
The language server was annoyed that these aren't actually parameters, they're members of an object that is the only parameter, so I just moved the docs below to make it happy.